src/main/kotlin/io/titandata/titan/providers/local/Tag.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2019 by Delphix. All rights reserved.
+ */
+
+package io.titandata.titan.providers.local
+
+import io.titandata.client.apis.CommitsApi
+
+class Tag (
+    private val commitsApi: CommitsApi = CommitsApi()
+) {
+    fun tagCommit(repository: String, commit: String, tags: List<String>) {
```

Comment:
Can we consistently use `guid: String` instead of `commit`?

Reply:
I guess I'm confused. Looking at Provider.kt, there are a half dozen commands that take a commit ID. They are all called "commit". The only exception is checkout(). I'm happy to leave that one as-is, but I see no reason why tag should be different than delete/push/pull/etc. They all take commit IDs and the parameter is named "commit" (along with the CLI option is "-c" for commit).

Reply:
You're right. I forgot we're in the provider scope here. It should be commit.

Reply:
For the CLI and the end user, it should definitely be commit. I prefer GUID in the provider side, I know I ran into confusion once and switched to GUID. I guess I didn't go back and refactor all the other instances of commit: String to guid: String in the providers. Small nit here.

Reply:
OK I'll leave it for now since it matches the others. I've gotten enough scope creep at this point, can come and refactor the others once we have end2end tests and can be a bit more confident making changes.